services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -1,26 +1,27 @@
-import { createContext, ReactNode, useMemo, useState } from "react";
+import { createContext, ReactNode, useEffect, useMemo, useState } from "react";
 // utils
 import {
   AnyObject,
   ReportDataShape,
   ReportDetails,
   ReportContextShape,
-  ReportStatusShape,
+  ReportShape,
 } from "types";
 import {
   getReportData,
   writeReportData,
   getReport,
-  getReportsByState,
   writeReport,
   deleteReport,
 } from "utils";
 // verbiage
 import { reportErrors } from "verbiage/errors";
 
 export const ReportContext = createContext<ReportContextShape>({
-  reportStatus: undefined as AnyObject | undefined,
+  report: undefined as AnyObject | undefined,
   reportData: undefined as AnyObject | undefined,
```

Comment:
Eventually we should firm these types up but I'm not concerned about it for right now.

Reply:
agreed